shared/css/color.scss

```diff
@@ -184,7 +184,7 @@ $facebook_brand_color: #3B5998;
 // ===----===----===---- Design system colors: ----===----===----===
 
 $light_primary_100: #ABDFE5;
-$light_primary_500: #0093A4;
+$light_primary_500: #0093a4;
```

Comment:
Should this stay as-is with the capitalized `A` in the hex code to match the others?

Reply:
Yeah, that's a good qustion.
I don't have a personal preference really, but in some of these set of files I encountered a style lint error that asked me to make hex code lower case so I made it the same in all the files.

Also, I see that actually in the begging of this file we already have lowercase hex codes so I think it's ok to have lowercase here